packages/create-vite/template-lit/src/index.css

```diff
@@ -1,8 +1,7 @@
 :root {
-  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
-  font-size: 16px;
-  line-height: 24px;
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   font-weight: 400;
+  line-height: 1.5;
```

Comment:
Could we move `line-height` above `font-weight` to keep it consistent with the other changes?

Reply:
Done!